README.md

```diff
@@ -173,19 +173,19 @@ jobs:
       - uses: actions/checkout@v2
         # these if statements ensure that a publication only occurs when
         # a new release is created:
-        if: ${{ steps.release.outputs.release_created }}
+        if: ${{ steps.release.outputs.releases_created }}
```

Comment:
I don't think this is correct: https://github.com/google-github-actions/release-please-action/blob/main/index.js#L56

Reply:
I think we were both mistaken - it looks like `releases_created` is actually set as output always: https://github.com/google-github-actions/release-please-action/blob/973b63d3d58b9a5a9fed0dd512501ad5a9ab319b/index.js#L49

Whereas `release_created` wasn't removed as I initially thought, but is only set if you are releasing a package at the root of the repo(?) Regardless, IMO it would be more correct to use the output that is always set in the example code to cover more use cases.